openshift-client/src/main/java/io/fabric8/openshift/client/OpenShiftLocalSubjectAccessReviewOperationsImpl.java

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.fabric8.openshift.client;
+
+import io.fabric8.kubernetes.client.Config;
+import io.fabric8.kubernetes.client.KubernetesClientException;
+import io.fabric8.kubernetes.client.dsl.Createable;
+import io.fabric8.kubernetes.client.dsl.Namespaceable;
+import io.fabric8.kubernetes.client.dsl.base.OperationContext;
+import io.fabric8.kubernetes.client.dsl.base.OperationSupport;
+import io.fabric8.openshift.api.model.DoneableLocalSubjectAccessReview;
+import io.fabric8.openshift.api.model.LocalSubjectAccessReview;
+import io.fabric8.openshift.api.model.SubjectAccessReviewResponse;
+import okhttp3.OkHttpClient;
+
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+
+public class OpenShiftLocalSubjectAccessReviewOperationsImpl extends OperationSupport implements Createable<LocalSubjectAccessReview, SubjectAccessReviewResponse, DoneableLocalSubjectAccessReview>, Namespaceable<OpenShiftLocalSubjectAccessReviewOperationsImpl> {
+  private String subjectAccessApiGroupName;
+  private String subjectAccessApiGroupVersion;
+  private String plural;
+
+  public OpenShiftLocalSubjectAccessReviewOperationsImpl(OkHttpClient client, Config config, String apiGroupName, String apiGroupVersion, String plural) {
+    this(new OperationContext().withOkhttpClient(client).withConfig(config), apiGroupName, apiGroupVersion, plural);
+    this.subjectAccessApiGroupName = apiGroupName;
+    this.subjectAccessApiGroupVersion = apiGroupVersion;
+    this.plural = plural;
```

Comment:
This is redundant, previous statement already takes care of this. I think you can also declare the fields final.